server/modules/analysis/api/entitiesApi.js

```diff
@@ -0,0 +1,36 @@
+import * as Request from '@server/utils/request'
+import * as AuthMiddleware from '@server/modules/auth/authApiMiddleware'
+
+import * as AnalysisService from '../service'
+
+export const init = (app) => {
+  // ====== GET - virtual_entities
+
+  app.get(
+    '/survey/:surveyId/virtual-entities/',
+    AuthMiddleware.requireRecordAnalysisPermission,
+    async (req, res, next) => {
+      try {
+        const { cycle, surveyId, offset, limit } = Request.getParams(req)
+        const list = await AnalysisService.fetchVirtualEntities({ surveyId, cycle, offset, limit })
```

Comment:
fetchVirtualEntities and countVirtualEntities should be added to NodeDefService.
(add those two methods in arena-core as well)